src/Serilog.Extensions.Logging/SerilogLoggerServicesExtensions.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.ComponentModel;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Serilog.Extensions.Logging;
+
+
+namespace Serilog
+{
+    /// <summary>
+    /// Extends <see cref="IServiceCollection"/> with Serilog configuration methods.
+    /// </summary>
+    public static class SerilogLoggerServicesExtensions
+    {
+        /// <summary>
+        /// Add Serilog to the logging pipeline.
+        /// </summary>
+        /// <param name="services">The <see cref="T:Microsoft.Extensions.DependencyInjection.IServiceCollection" /> to add services to.</param>
+        /// <param name="logger">The Serilog logger; if not supplied, the static <see cref="Serilog.Log"/> will be used.</param>
+        /// <returns>The logger factory.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static IServiceCollection AddSerilog(this IServiceCollection services, ILogger logger)
```

Comment:
This method can be removed - the reason for it appearing in `SerilogLoggerFactoryExtensions` is backwards binary compatibility.